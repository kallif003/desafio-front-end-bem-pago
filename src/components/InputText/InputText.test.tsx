import * as React from "react"
import { render } from "@testing-library/react"
import { InputText } from "./"

describe("Input Text components", () => {
	it("Renders Input Text components", () => {
		const { container } = render(<InputText>Test</InputText>)
		expect(container.firstChild).toMatchSnapshot()
	})
})
