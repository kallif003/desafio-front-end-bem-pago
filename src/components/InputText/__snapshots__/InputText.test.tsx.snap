// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Text components Renders Input Text components 1`] = `
<textarea
  class="sc-bdvvtL bJjdMu"
  cols="80"
  name="inputText"
  placeholder="Digite aqui suas observações:"
  rows="10"
>
  Test
</textarea>
`;
